00XX:reassert-fee-payment-method

```diff
@@ -0,0 +1,54 @@
+## HIP: Driving Value Into The IOT Network Through IOT Token Payments
```

Comment:
Suggestion to add in the title that this is about assert fees.

would suggest something like:
Changing Assert Fee Denomination from $DC to $IOT to Drive Value Into The IOT Network